lib/bolt/cli.rb

```diff
@@ -471,7 +472,10 @@ def execute(options)
         end
 
         outputter.print_summary(results, elapsed_time)
+        successful = results.values.all?(&:success?)
```

Comment:
Why wasn't `results.ok` sufficient here?

Reply:
Oh, because it's a hash, not an execution result yet. It might help to highlight what we expect `results` to be when we get here.

Reply:
I expect that [BOLT-286](https://tickets.puppetlabs.com/browse/BOLT-286) will have an impact on this.